merlin-server/sql/merlin/tables/mission_model.sql

```diff
@@ -72,3 +79,18 @@ create trigger increment_revision_on_update_mission_model_jar_trigger
 after update on uploaded_file
 for each row
 execute function increment_revision_on_update_mission_model_jar();
+
+create function mission_model_default_description()
```

Comment:
🔶 I'm curious about the need for this default description - it doesn't seem to add any new information, and costs us another trigger to maintain. Could we consider making this a column with a default empty string value?

Reply:
Done